python/instrumentation/openinference-instrumentation-langchain/src/openinference/instrumentation/langchain/_tracer.py

```diff
@@ -105,17 +152,15 @@ def _start_trace(self, run: Run) -> None:
         # leaving all future spans as orphans. That is a very bad scenario.
         # token = context_api.attach(context)
         with self._lock:
-            self._runs[run.id] = _Run(span=span, context=context)
+            self._spans[run.id] = span
```

Comment:
this is a bit confusing from a naming perspective. Maybe `_spans_by_run` would make the Dict keys more clear

Reply:
Anyway, i changed it for the extra clarity.